js/utils.js

```diff
@@ -10,6 +10,8 @@ export const usbPortFilters = [
     { usbVendorId: 0x0403, usbProductId: 0x6001 }, /* FT232R */
 ];
 
+export const consoleBaudrateC2 = 74880; // supported console baudrate for C2
```

Comment:
Change the comment - supported console baudrate for chips with 26MHZ frequency.

Reply:
Updated the comment targeting 26MHz frequency chips.